esmvaltool/references/zhang11wcc.bibtex

```diff
@@ -0,0 +1,13 @@
+@article{zhang11wcc,
+	doi = {10.1002/wcc.147},
+	url = {https://doi.org/10.1002%2Fwcc.147},
+	year = 2011,
+	month = {oct},
+	publisher = {Wiley},
+	volume = {2},
+	number = {6},
+	pages = {851--870},
+	author = {Xuebin Zhang and Lisa Alexander and Gabriele C. Hegerl and Philip Jones and Albert Klein Tank and Thomas C. Peterson and Blair Trewin and Francis W. Zwiers},
+	title = {Indices for monitoring changes in extremes based on daily temperature and precipitation data},
+	journal = {Wiley Interdisciplinary Reviews: Climate Change}
+}
```

Comment:
Newline is missing in all files, I don't whether this could be a problem.

Reply:
a newline is added at the end of each file.